src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -88,6 +94,32 @@ SHOULD be present: For consistency between studies and institutions, we
 encourage users to extract the values of these fields from the actual raw data.
 Whenever possible, please avoid using ad-hoc wording.
 
+{{ MACROS___make_metadata_table(
+   {
+      "InstitutionName": "RECOMMENDED",
```

Comment:
This contains the following:

> Corresponds to DICOM Tag 0008, 0080 InstitutionName.

I think that sentence should not be part of MEG (EEG, ...) because it is only relevant to NIfTI files. Can this be easily achieved?

Reply:
In fact, this problem is also present for many of the fields below: Information about DICOM tags feels misplaced in the non-NIfTI part of the spec.

Reply:
I can make these section-specific additions.

Reply:
Cross-referencing the set of fields with DICOM Tags mentioned in the definitions with those that are used in non-NIFTI parts of the spec is going to take a bit of work. Would you rather I (1) moved the DICOM mentions to section-specific definitions for _all_ fields mentioning DICOM Tags (which would future-proof the definitions, but would probably not be necessary for many MRI-specific fields) or (2) identify only fields used in non-NIFTI sections and fix those (which will take more work but will keep the MRI-specific field definitions specific)?

Reply:
Good question, initially (and practically) I wanted your suggestion 2, because it doesn't make sense for me to read about DICOM tags in an EEG specification (it's confusing in that context). I am not entirely sure I understand your suggestion 1, but maybe it'd be most consistent to purge dicom information from the schema definitions, and "append" them to the render-blocks like you currently append the info on "optional", "required", etc.?

I feel like more people should offer their opinion on this.

Reply:
For example, here are the new "Manufacturer" cells in tables within MRI and physio data, respectively:

### MRI
![Screen Shot 2021-06-16 at 12 03 29 PM](https://user-images.githubusercontent.com/8228902/122254323-04c1e200-ce9b-11eb-926b-99149f5c5717.png)

### Physio
![Screen Shot 2021-06-16 at 12 03 45 PM](https://user-images.githubusercontent.com/8228902/122254339-07bcd280-ce9b-11eb-9f42-db44d69606f8.png)

Reply:
great, thanks - this is exactly what I hoped for. If it's not too much of a hassle, I would be +1 to do this consistently for all metadata fields.

Reply:
This is probably just me being lazy, but I think that it would end up being easier to fix new conflicts (e.g., allowing some MRI/PET-specific fields for other modalities) _after_ this PR is merged, rather than trying to get ahead of the problem now. Just because the only remaining tables will be affected and folks will be able to see the problem in their PRs.

One thing worth noting: The table-specific appended description text is just a temporary fix. We will ultimately want to directly include that information in the schema- perhaps through injectable YAML files?

Reply:
> just because the only remaining tables will be affected and folks will be able to see the problem in their PRs.

right, fine with me. Thanks!

> We will ultimately want to directly include that information in the schema- perhaps through injectable YAML files?

yes, makes sense.